docs: add MCP server setup instructions

clydin opened the pull request:
Adds a new section to the "Develop with AI" documentation page that details how to configure the Angular CLI's Model Context Protocol (MCP) server.

This section includes:
- Instructions on how to use the command.
- IDE-specific JSON configurations for VS Code, JetBrains, Firebase Studio, and Cursor.
- Clarification on the different JSON structures required by each IDE.

MarkTechson reviewed (changes requested):
This is great. Added some proposed wording and also I suggest that we move this to it's own page.

clydin committed:
docs: create MCP server setup guide

Introduces a new guide for setting up the Angular CLI's Model Context Protocol (MCP) server.

The guide includes:
- Instructions on how to use the command.
- IDE-specific JSON configurations for VS Code, JetBrains, Firebase Studio, and Cursor.
- Clarification on the different JSON structures required by each IDE.

MarkTechson reviewed (approved):
LGTM! 

Thanks, Charles! Developers are really going to benefit here

kirjs commented:
This PR was merged into the repository by commit 081d73907d92a2b850861e3d5d8884087a0f11df.

The changes were merged into the following branches: main, 20.1.x
